inc/pantheon-multisite-finalize.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/*
+ * If a site has multisite enabled, but has not had the final installation
+ * steps completed, alert the user and provide links.
+ */
+
+function pantheon_multisite_install_finalize_message() {
+	?>
+	<div class="notice notice-info is-dismissible">
+		<p>Your WordPress Site Network is almost ready!</p>
+		<p>Visit <a href="https://pantheon.io/docs/guides/multisite/config/#install-the-wordpress-site-network">Pantheon WordPress Site Network Configuration</a>
```

Comment:
```suggestion
		<p><?php echo wp_kses_post( sprintf( __( 'Visit <a href="%s">Pantheon WordPress Site Network Configuration</a> for documentation on how to finalize configuration of your site network.', 'pantheon-systems' ), 'https://pantheon.io/docs/guides/multisite/config/#install-the-wordpress-site-network' ) ); ?></p>
```

See [pantheon-updates.php](https://github.com/pantheon-systems/pantheon-mu-plugin/blob/main/inc/pantheon-updates.php#L74) for more usage examples.

Reply:
@jspellman814 I don't mind adding it at all, but I'm curious about one thing. `wp_kses_post` functions to strip bad tags, but given that this is hard coded content which must be changed in code there's no chance of bad data being in there. With that in mind, what do we need the `wp_kses_post` for?

cc @CdrMarks because it came up in a 1-1

Reply:
Just a force of habit on my part 😅 we can do without it in this case, so the suggestion can be changed to:

```suggestion
		<p><?php echo sprintf( __( 'Visit <a href="%s">Pantheon WordPress Site Network Configuration</a> for documentation on how to finalize configuration of your site network.', 'pantheon-systems' ), 'https://pantheon.io/docs/guides/multisite/config/#install-the-wordpress-site-network' ); ?></p>
```

Reply:
No worries, it wouldn't have been harmful at all, I was just curious if the function did more than I knew or if there was a WordPress'ism that I didn't know.